src/main/resources/component-kernel.conf

```diff
@@ -109,17 +109,44 @@ cache {
 }
 
 http.outcall {
-    # Outcall (and also XMLCall and SOAPClient) use an internal mini-blacklist. Each host which ran into a
-    # connect timeout, is put onto a "blacklist" and no further connections will be attempted for the given
-    # amount of time as defined here.
-    #
-    # This hopefully prevents "cascading" failures e.g. if a page load itself triggers multiple requests
-    # (think of a shop system which tries to fetch prices for multiple items), the connect timeout might
-    # add up pretty quickly. Therefore, we only try once and then stop trying so that we can render the
-    # remaining part of the site (or at least fail gracefully).
+
+    # Contains the default timeouts. "default" will take effect in every call in which no other timeout is specified.
     #
-    # Set this value to 0 to disable blacklisting.
-    connectTimeoutBlacklistDuration = 10s
+    # This timeouts can be expanded by new blocks in dependant projects. For the configuration to take effect, on an
+    # outcall the sirius.kernel.xml.Outcall.withConfiguredTimeout needs to be called with the defined block
+    # key as parameter.
+    timeouts {
+
+        # Outcall (and also XMLCall and SOAPClient) use an internal mini-blacklist. Each host which ran into a
+        # connect timeout, is put onto a "blacklist" and no further connections will be attempted for the given
+        # amount of time as defined here.
+        #
+        # This hopefully prevents "cascading" failures e.g. if a page load itself triggers multiple requests
+        # (think of a shop system which tries to fetch prices for multiple items), the connect timeout might
+        # add up pretty quickly. Therefore, we only try once and then stop trying so that we can render the
+        # remaining part of the site (or at least fail gracefully).
+        #
+        # Set this value to 0 to disable blacklisting.
+        connectTimeoutBlacklistDuration = 10s
```

Comment:
Why not scope the blacklist duration as well?

Reply:
... or put it outside of `timeouts`; it is actually not a timeout, but the blacklisting duration. Which also would avoid breaking existing config

Reply:
the black list works on url basis. But config key was moved to make pr non-breaking.